test/e2e/cluster_size_autoscaling.go

```diff
@@ -137,14 +137,24 @@ var _ = framework.KubeDescribe("Cluster size autoscaling [Slow]", func() {
 	})
 
 	It("should correctly handle pending and scale down after deletion [Feature:ClusterSizeAutoscalingScaleDown]", func() {
-		By("Small pending pods increase cluster size")
-		ReserveMemory(f, "memory-reservation", 100, nodeCount*memCapacityMb, false)
-		// Verify, that cluster size is increased
+		By("Manually increase cluster size")
+		increasedSize := 0
+		newSizes := make(map[string]int)
+		for key, val := range originalSizes {
+			newSizes[key] = val + 2
+			increasedSize += val + 2
+		}
+		restoreSizes(newSizes)
```

Comment:
Will do, in a next pr together with some refactoring (move cleanup out of tests, etc)